collections/_events/2023-10-26-summit-nescala.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: event
+
+title: "Typelevel Summit NEScala"
+short_title: "Typelevel Summit"
+date_string: "October 26, 2023"
+location: "Online"
+description: "One day of recorded talks and one day of unconference, co-located with NE Scala."
```

Comment:
```suggestion
description: "One day of talks and one day of unconference, co-located with NE Scala."
```
Calling it "recorded talks" makes it sounds like the talks will all be pre-recorded, which I don't think is true -- while we're going to allow for pre-recorded, I expect most to be live, and for some to have Q&A.  Tweaked the section below to clarify (I think).